src/System.Private.ServiceModel/tests/Common/Infrastructure/src/IUtil.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Reviewers: to give some context, the IUtil interface was embedded in in ServiceInterfaces available only in the Common\Scenarios project (which depends on this Common\Infrastructure project, which would create a circular dependency).  To follow the pattern we found worked best for the Bridge "helper" service interactions, I have moved this interface as well as the certificate and endpoint helper methods into this Common\Infrastructure project.  They have not been modified -- only rehomed.